Jellyfin.Api/Helpers/HlsCodecStringHelpers.cs

```diff
@@ -121,5 +122,23 @@ public static string GetEAC3String()
         {
             return "mp4a.a6";
         }
+
+        /// <summary>
+        /// Gets an FLAC codec string.
+        /// </summary>
+        /// <returns>FLAC codec string.</returns>
+        public static string GetFLACString()
+        {
+            return "fLaC";
+        }
+
+        /// <summary>
+        /// Gets an ALAC codec string.
+        /// </summary>
+        /// <returns>ALAC codec string.</returns>
+        public static string GetALACString()
+        {
+            return "alac";
+        }
```

Comment:
These might be better as constants unless you expect them to change depending on the platform or something.